src/lib/fragments/UkGovDashComponents.react.js

```diff
@@ -1,41 +0,0 @@
-import React, {Component} from 'react';

```

Comment:
Why is this file no longer needed? (but the fragment is?)

Reply:
Because it qas the base file that was generated when you use the dash react generator

Reply:
Good to know! Thanks Jordan

Reply:
Doh, I see what you mean.  Yeah I thought there wasn't a component when I checked.  How wrong I was.